### PR TITLE
deleting excessed free spaces

### PR DESCRIPTION
**Pull request by anna-dao**
just deleted some free spaces to make the file more accurate.

### CODE REVIEW

**Comment by suchov** [comment]
Yes, but also add some free spaces??? :D
Can you pls check the lines changed?

File: README.md [hunk 1]
```diff
@@ -23,32 +23,30 @@ Good luck, have a good day, success in your studies, thank you Kottans!
 
 :love_letter::heart::smile_cat::mechanical_arm::woman_technologist:
 
-
```
Comment (suchov) [inline README.md:26]: no need to do this, pls leave only the desired lines editing

**Review by suchov — CHANGES_REQUESTED** [review]

**Comment by anna-dao** [comment]
seems I added some free spaces too, will delete them and pull it one more time.. 
sorry!!

**Comment by suchov** [comment]
No worries, it's a play with file at the end of the day ;)

**Pull request closed without merging**